djautotask/api.py

```diff
@@ -229,15 +235,15 @@ def create_object(entity_type, entity_fields):
     return at.create(entity_object).fetch_one()
 
 
-def update_assigned_resource(at_object, resource, role):
-    entity = at_object.__class__.__name__
-    query = Query(entity)
-    query.WHERE('id', query.Equals, at_object.id)
+def delete_objects(entity_type, objects):
+    """
+    Make a request to Autotask to delete the given Autotask entities.
+    """
     at = init_api_connection()
+    rename_this_list = []
```

Comment:
Should this be renamed!? :smile:

Reply:
Nah, you know what, its fine like that. /s